src/client/components/app/Transactions/Labs/LabDepositTx.tsx

```diff
@@ -64,6 +68,10 @@ export const LabDepositTx: FC<LabDepositTxProps> = ({ onClose }) => {
     : userTokens;
   const sellTokensOptionsMap = keyBy(sellTokensOptions, 'address');
   const selectedSellToken = sellTokensOptionsMap[selectedSellTokenAddress ?? ''];
+  const [tokenAllowance, isLoadingTokenAllowance, getTokenAllowanceErrors] = useAllowance(
```

Comment:
```suggestion
  const [tokenAllowance, isLoadingTokenAllowance, tokenAllowanceError] = useAllowance(
```
Lets remove the `get` since it makes it look like if its a function

Reply:
solved on 6ad01db108237880b69ecb49695a74b419be2063